index.html

```diff
@@ -38,8 +41,190 @@ <h1>I'm Mucha <br> Glad to see <br> you!</h1>
+            </ul>
+
+            <p class="tonic-def">A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups
+                required.</p>
+
+            <ul class="stack">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="js">javaScript</li>
+            </ul>
+
+            <div class="button">
+                <button type="button">See Project</button>
+            </div>
+        </div>
+    </div>
+
+    <!-- card 3 -->
+    <div class="work-container3">
+        <div class="portfolio-img">
+            <img src="images/Snapshoot-Portfolio-3.png" alt="">
+        </div>
+
+        <div class="work-content">
+
+
+            <h1 class="tonic">Tonic</h1>
+
+            <ul class="prop">
+                <li class="can">CANOPY</li>
+                <li class="bac"><img src="images/Counter.png" alt=""> Back End Dev</li>
+                <li class="num"><img src="images/Counter.png" alt=""> 2015</li>
+            </ul>
+
+            <p class="tonic-def">A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups
+                required.</p>
+
+            <ul class="stack">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="js">javaScript</li>
+            </ul>
+
+            <div class="button">
+                <button type="button">See Project</button>
+            </div>
+        </div>
+    </div>
+
+    <!-- card-4 -->
+    <div class="work-container4">
+        <div class="portfolio-img">
+            <img src="images/Snapshoot-Portfolio-4.png" alt="">
+        </div>
+
+        <div class="work-content">
+
+
+            <h1 class="tonic">Multi-Post<br>Stories</h1>
+
+            <ul class="prop">
+                <li class="can">CANOPY</li>
+                <li class="bac"><img src="images/Counter.png" alt=""> Back End Dev</li>
+                <li class="num"><img src="images/Counter.png" alt=""> 2015</li>
+            </ul>
+
+            <p class="tonic-def">A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups
+                required.</p>
+
+            <ul class="stack">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="js">javaScript</li>
+            </ul>
+
+            <div class="button">
+                <button type="button">See Project</button>
+            </div>
+        </div>
     </div>
 
+    <!-- about me  -->
+    <div class="introd">
+        <h1>About<br> Myself</h1>
+        <p>I'm a software developer! I can help you <br> build a product, feature or website Look <br> through some
+            of my work and <br> experience! If you like what you see and <br> have a project you need coded, don't
+            <br> hestiate to contact me
+        </p>
+
+        <a href="#" class="con">LET'S CONNECT</a>
```

Comment:
- [ ] You have done great by ensuring that the heading uses `a` HTML tag 👏 . However, in this section let's consider using `h2-h6`. This is because to be [Semantically](https://www.w3schools.com/html/html5_semantic_elements.asp) correct, the page should have only one `h1` which is the main heading of the page. In addition, doing this increases the accessibility of the page 👍 .